poetry/utils/_compat.py

```diff
@@ -23,6 +23,11 @@
 except ImportError:
     import urlparse
 
+try:
+    from os import cpu_count
+except ImportError:
```

Comment:
```suggestion
except ImportError:  # Python 2
```
Jus so that we can clean this up later

Reply:
Done :-)